config/autoload/home-page-content.global.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+// phpcs:disable Generic.Files.LineLength.TooLong
+
+return [
+    'commercial-vendors'        => [
+        [
+            'name' => 'Roave',
+            'logo' => '/images/vendors/roave.png',
+            'url'  => 'https://roave.com',
+            'text' => 'Roave is a full-service web development firm, offering consulting, training, software development, and more.',
+        ],
+        [
+            'name' => 'Zend by Perforce',
+            'logo' => '/images/vendors/zend.png',
+            'url'  => 'https://www.zend.com/products/laminas-enterprise-support',
+            'text' => 'Zend is "The PHP Company", and offers training, services, and production support for Laminas and other PHP applications.',
+        ],
+        [
+            'name' => 'Apidemia',
+            'logo' => '/images/vendors/apidemia.png',
+            'url'  => 'https://www.apidemia.com',
+            'text' => 'Apidemia is a web development company, offering software development and consulting. Since 2005, we have focused on the PHP ecosystem, and our tools of trade have been Zend Framework 1, 2, and 3, and then Laminas and Mezzio. We created DotKernel, a collection of PSR-7 Middleware applications built on top of the Mezzio microframework and Laminas components.',
+        ],
+    ],
+    'projects-using-components' => [
+        [
+            'name' => 'Magento',
+            'logo' => '/images/sponsors/magento.png',
+        ],
+        [
+            'name' => 'Owncloud',
+            'logo' => '/images/sponsors/owncloud.png',
+        ],
+        [
+            'name' => 'Drupal',
+            'logo' => '/images/sponsors/drupal.png',
```

Comment:
The Drupal logo should also contain the wordmark: https://www.drupal.org/about/media-kit/logos